src/main/java/geometry/Rectangular.java

```diff
@@ -0,0 +1,24 @@
+package geometry;
+
+public abstract class Rectangular implements Shape {
+    private double sideA;
+    private double sideB;
+
+    public Rectangular(double sideA, double sideB) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    public Rectangular(double sideA) {
```

Comment:
please move this constructor to Square class, rectangular should have two sides as arguments for one constructor (please do the same for Round class as well)

Reply:
done